modules/nf-core/sentieon/bwamem/main.nf

```diff
@@ -21,15 +24,17 @@ process SENTIEON_BWAMEM {
     task.ext.when == null || task.ext.when
 
     script:
-    // Exit if running this module with -profile conda / -profile mamba
-    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
-        error "Sentieon modules do not support Conda. Please use Docker / Singularity / Podman instead."
-    }
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def sentieon_auth_mech_base64 = task.ext.sentieon_auth_mech_base64 ?: ''
     def sentieon_auth_data_base64 = task.ext.sentieon_auth_data_base64 ?: ''
 
+    if (workflow.containerEngine == 'singularity') {
+        fix_ld_library_path = 'LD_LIBRARY_PATH=/usr/local/lib/:\$LD_LIBRARY_PATH;export LD_LIBRARY_PATH'
+    } else {
+        fix_ld_library_path = ''
+    }
+
```

Comment:
I'll put that at the top of the script part, where we had the conda bit.
And can you add a comment about what is happening there.
And why